akka-actor-testkit-typed/src/test/scala/docs/akka/actor/testkit/typed/scaladsl/AsyncTestingExampleSpec.scala

```diff
@@ -24,10 +32,27 @@ object AsyncTestingExampleSpec {
     }
   }
   //#under-test
+
+  //#under-test-2
+  case class Message(i: Int, replyTo: ActorRef[Try[Int]])
+
+  class Producer(publisher: ActorRef[Message])(implicit scheduler: Scheduler) {
+
+    def produce(messages: Int)(implicit timeout: Timeout): Unit = {
+      (0 until messages) foreach publish
+    }
+
+    private def publish(i: Int)(implicit timeout: Timeout): Future[Try[Int]] = {
+      publisher ? ((ref: ActorRef[Try[Int]]) ⇒ Message(i, ref))
```

Comment:
This doesn't compile as `Message(i, _)` or `ref => Message(i, ref)`? I thought that was fixed in the latest Scala releases...

Reply:
Thanks for the hint, my IDEs scala sdk was outdated ;-)

`Message(i, _)` though also fails in sbt with
```
[error] AsyncTestingExampleSpec.scala:46:30: missing parameter type for expanded function ((x$2: <error>) => publisher.$qmark(Message(i, x$2)))
[error]       publisher ? Message(i, _)
[error]                              ^
```
but `ref ⇒ Message(i, ref)` works.